csfieldguide/static/interactives/no-help/scss/no-help.scss

```diff
@@ -0,0 +1,6 @@
+#interactive-no-help {
+    background-color: #4caf50;
```

Comment:
Rather than set the background, border and text colours, you can use `btn-success` instead of `btn-primary` to get the green button with white text.
See: https://getbootstrap.com/docs/4.1/components/buttons/